plugins/amdgpu/Makefile

```diff
@@ -9,14 +9,15 @@ include $(__nmk_dir)msg.mk
 
 CC      		:= gcc
 PLUGIN_CFLAGS  		:= -g -Wall -Werror -D _GNU_SOURCE -shared -nostartfiles -fPIC
+PLUGIN_LDFLAGS		:= -lrt -lpthread
```

Comment:
This seems unrelated to the shared memory changes. Should this be part of the commit that adds multi-threaded saving of VRAM?

Reply:
the shm_* functions need -lrt, and the sem_* functions need -lpthread.
The multi-threaded saving of VRAM should need:
+PLUGIN_LDFLAGS		:= -lpthread

I will add that to the multi-threaded commit if it is needed.